features/step_definitions/app_steps.rb

```diff
@@ -85,6 +85,10 @@
     step 'I sign in as "alice@example.com" with password "12345678"'
 end
 
+Given /^I am on topic ID (.*)/ do |topic_ID|
+    visit "/#/questions/#{topic_ID}"
```

Comment:
this shouldn't pass anymore... you'll need to manually navigate to the appopriate page using button presses.